scripts/build.py

```diff
@@ -380,7 +491,7 @@ def hash_should_be_inserted(filepath):
                    in FILEPATHS_NOT_TO_RENAME)
 
 
-def copy_files_source_to_target(source, target, file_hashes):
+def copy_files_source_to_target(source, target, file_hashes, copy_tasks):
```

Comment:
Probably wants a different name that doesn't imply that this function is actually doing file copying.

Reply:
Renamed function `copy_files_source_to_target()` to `generate_copy_tasks_to_copy_from_source_to_target()`